### ISSUE TITLE
OLAC Challenges

### ISSUE THREAD

**Issue by HughP** [HughP/LinguisticsLibrarianList#1]
@RichardLitt did you have some comments about the utility of OLAC? 

I am starting a draft here: https://github.com/HughP/LinguisticsLibrarianList/blob/master/OLACChallenges.md
This open document came about due to discussions here: https://github.com/RichardLitt/endangered-languages/issues/44

**Comment by cesine** [comment]
maybe you can hold off on bashing OLAC? or talk to the OLAC guys first directly? a little birdy told me that their website hasn't been updated since Thr 24 Feb 2011 14:40:37 EST (see screenshot) maybe they know something you don't, and you wont know know unless you talk to them straight.

![screen shot 2015-03-05 at 4 08 31 pm](https://cloud.githubusercontent.com/assets/196199/6502010/e8e41c34-c351-11e4-9cdc-97a4885d99bb.png)

**Comment by HughP** [comment]
Actually, @cesine  have been in touch with Steven Bird (I met him for the first time at ILDC4, but had been in touch with him previously via email), and the core text on that shared document was shared in another format with Gary Simons about 6 months ago. There are two things which are "OLAC" in people's minds:
1. a [metadata standard/schema](http://www.language-archives.org/tools.html) for interoperability
2. a [web presentation for finding resources](http://search.language-archives.org/index.html) (based on said metadata standard)

If I understand the site with your screenshot, that update date is for a unique html page, and is likely an html 3 or 4 Doctype, querying the server much like I used to do with SHTML. You might find some other pages with a much more recent update date. And the content of the web presentation is continually running and updating via a cron job. So, there is more recent activity than 2011, its just that they are not using a CMS to show it... is that birdy you are referencing a person or are you making reference to the code?

I don't see myself as "bashing" OALC, only recognizing that it is a foundation which needs to be built on. Gary has personally (to me) advocated that OLAC as a service provider is strictly an aggregator.  His example for years has been "Google", but if we follow Google and the market place trends then we come to realize that the role of aggregators are limited in web society and that they need help with relevance, and with social graph information. OLAC has neither of these.

To be clear, I don't have a "problem" with the OLAC metadata standard/schema, only that some institutions seem to not use it or provide XML feeds when they could/should. I think it is extremely helpful in the linguistics industry. I also don't think that the OLAC metadata is the only metadata that an institution needs to use, which most institutional repository managers would also recognize as being true. Though from time to time the OLAC terminology taxonomies should be updated.

**Comment by RichardLitt** [comment]
I don't have a problem with OLAC - it's a useful resource. I think it just geared more towards linguists, I wonder about their discoverability, and it doesn't seem to archive resources (as mentioned, it is just an aggregator).

**Comment by HughP** [comment]
@RichardLitt in response to:

> it doesn't seem to archive resources (as mentioned, it is just an aggregator).

It doesn't even keep a record of the XML files it aggregates. for example, It doesn't do a diff check and display new files as being "new additions".

**Comment by RichardLitt** [comment]
So, that's something that I would suggest we (or someone) do, basically.

**Comment by HughP** [comment]
I agree. I have been contemplating a move in that direction for several years, but have lacked the social encouragement to do so. I suggest using Drupal as a public facing interface, but there still needs to be a discussion about data storage and architecture.
